CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Notable changes.
 
 ## November 2022
 
+### [0.25.1]
```

Comment:
Do you want to create a separate PR to track this later on? or maybe just change the title of PR.

Reply:
I'll change the title 👍